functions/app/index.html

```diff
@@ -255,29 +248,6 @@ <h2 class="mb-4">The buttons below are impossible to resist...</h2>
 
   <!-- Custom scripts for this template -->
   <script src="js/stylish-portfolio.min.js"></script>
-
```

Comment:
This code is boiler plate code and only intended to display what Firebase services were being used, since we are now using Firebase Functions its all just redundant code.